sdk/storage/tests.yml

```diff
@@ -6,8 +6,7 @@ extends:
     ServiceDirectory: storage
     BuildInParallel: true
     TimeoutInMinutes: 180
-    Location: canadacentral

```

Comment:
Did this move somewhere else? we shouldn't leave canadacentral for storage.
Also, some of the preview features won't work in UsGov and China right away. How do we filter tests that should run there?

(this one is important)

Reply:
@tzhanl is the issue with here that it doesn't apply for multiple cloud overrides? If so, the current way to deal with that is to pass in the entire CloudConfig parameter, like:

```
CloudConfig:
  UsGov:
    SubscriptionConfiguration: $( pipeline variable... )
    Location: usgovvirginia
  Public:
    SubscriptionConfiguration: $( pipeline variable... )
    Location: canadacentral
```

Reply:
@kasobol-msft for the latter, this is an important point. I don't have a great answer for this yet, with the current state of the pipeline. I'm working on a larger overhaul to support better definition of test matrixes with parameters, so I'm thinking implementing this functionality on top of that change would make sense, and keeping it disabled until then for sovereign clouds.